tests/e2e/util.go

```diff
@@ -1835,7 +1835,7 @@ func connectESX(username string, addr string, cmd string) (string, error) {
 }
 
 // getPersistentVolumeSpecWithStorageclass is to create PV volume spec with given FCD ID, Reclaim Policy and labels
-func getPersistentVolumeSpecWithStorageclass(volumeHandle string, persistentVolumeReclaimPolicy v1.PersistentVolumeReclaimPolicy, storageClass string, labels map[string]string) *v1.PersistentVolume {
+func getPersistentVolumeSpecWithStorageclass(volumeHandle string, persistentVolumeReclaimPolicy v1.PersistentVolumeReclaimPolicy, storageClass string, labels map[string]string, diskSize string) *v1.PersistentVolume {
```

Comment:
same here, this declaration of `diskSize` can be easily confused with `diskSize` from e2e _common.go

Reply:
renamed to sizeOfDisk